doc/whats_new/v1.5.rst

```diff
@@ -113,6 +109,17 @@ Changelog
   By default, the functions will retry up to 3 times in case of network failures.
   :pr:`28160` by :user:`Zhehao Liu <MaxwellLZH>` and :user:`Filip Karlo Došilović <fkdosilovic>`.
 
+:mod:`sklearn.decomposition`
+............................
+
+- |Enhancement| :class:`compose.TransformedTargetRegressor` now raises an error in `fit` if
+  only `inverse_func` is provided without `func` (that would default to identity) being
+  explicitly set as well. :pr:`28483` by :user:`Stefanie Senger <StefanieSenger>`.
```

Comment:
This should be under the compose module right? You included it under the decomposition module somehow.

Reply:
Indeed, this should be under the compose section and keep the new entry in the decomposition section.